libs/vscode/nx-cli-quickpicks/src/lib/select-affected-flags.ts

```diff
@@ -0,0 +1,106 @@
+import { OptionType, Option } from '@nx-console/shared/schema';
+import { selectFlags } from './select-flags';
+
+/**
+ *
+ */
```

Comment:
Was something important missing here?

Reply:
nah this empty comment block was there before I just copied it over.